sdk/eventhub/event-hubs/README.md

```diff
@@ -35,87 +35,99 @@ npm install @types/node
+
+ - A **consumer group** is a view of an entire Event Hub. Consumer groups enable multiple consuming applications to each have a separate view of the event stream, and to read the stream independently at their own pace and from their own position.  There can be at most 5 concurrent readers on a partition per consumer group; however it is recommended that there is only one active consumer for a given partition and consumer group pairing. Each active reader receives all of the events from its partition; If there are multiple readers on the same partition, then they will receive duplicate events. 
+
+ For more concepts and deeper discussion, see: [Event Hubs Features](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features)
+
+ 
 ### Authenticate the client
 
 Interaction with Event Hubs starts with an instance of the [EventHubClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest) class. You can instantiate
-this class using one of the below static methods on it
-
-- [createFromConnectionString](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubclient?view=azure-node-latest#createfromconnectionstring-string--string--clientoptions-)
-  - This method takes the connection string and entity name to your Event Hub instance. You can get the connection string
-    from the Azure portal
-- [createFromTokenProvider](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromtokenprovider-string--string--tokenprovider--clientoptionsbase-)
-  - This method takes the host name and entity name of your Event Hub instance and your custom Token Provider. The
-    host name is of the format `name-of-event-hub-instance.servicebus.windows.net`.
-- [createFromAADTokenCredentials](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromaadtokencredentials-string--string--applicationtokencredentials---usertokencredentials---devicetokencredentials---msitokencredentials--clientoptionsbase-)
-  - This method takes the host name and entity name of your Event Hub instance and a credentials object that you need
-    to generate using the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth)
-    library. The host name is of the format `name-of-event-hub-instance.servicebus.windows.net`.
+this class using one of the below
+
+- [EventHubClient](link)
```

Comment:
Even if we add the link here, it would look weird to have just the word `EventHubClient` here. Maybe a code snippet would be better?

`const client = new EventHubClient("my-connection-string", "my-event-hub")` ?